docs/src/pages/styles/basics/basics.md

```diff
@@ -1,8 +1,13 @@
 # @material-ui/styles
 
-<p class="description">You can use Material-UI's styling solution in your app, whether or not you are using Material-UI components.</p>
+<p class="description">The legacy styling solution of Material-UI.</p>
 
-> **Note**: `@material-ui/styles` is the _**legacy**_ styling solution for Material-UI. It is deprecated in v5. It depends on [JSS](https://cssinjs.org/) as a styling solution, which is not used in the `@material-ui/core` anymore. If you don't want to have both emotion & JSS in your bundle, please refer to the [`@material-ui/system`](/system/basics/) documentation which is the recommended alternative.
+> ⚠️ `@material-ui/styles` is the legacy styling solution for Material-UI.
+> It's **deprecated** in v5, which is not used by the other packages anymore.
```

Comment:
That sentence is grammatically off. This should probably be split into two sentences: one that states deprecated, the other that we no longer use it internally. These two facts are unrelated.

Reply:
Ok, I'm keeping the previous wording, it was great.